egs2/TEMPLATE/asr1/pyscripts/feats/ssl_feature_utils.py

```diff
@@ -97,7 +98,15 @@ def __init__(self):
 
     def load_audio(self, path: str, ref_len: Optional[int] = None):
         wav, sr = sf.read(path)
-        assert sr == self.sample_rate, sr
+        # assert sr == self.sample_rate, sr
```

Comment:
sorry I may miss it, but is there specific reason that you would like to comment out the line?

Reply:
I think if the audio sample rate(sr) is different from the sample rate, then this assertion cannot pass. But the mismatch is the reason we added the audio resample here?  (I followed PR#5795)